src/components/TasksLeft.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
+import Pluralize from 'react-pluralize';
 
 class TasksLeft extends React.Component {
 
-    render() {       
-        return <h1 styles={styles}>You have {this.props.tasksToDo.length} {this.props.tasksToDo.length !== 1 ? "tasks" : "task"} remaining!</h1>
+    render() { 
+        
+        // Work out how many of the tasks are left to complete
+        let tasksRemaining = 0;
+        for(let task of this.props.tasks) {
+            if(task.completed === false) {
+                tasksRemaining++;
+            }
+        }
+
+
+
+        return <h1 styles={styles}>You have <Pluralize singular={'task'} zero={"no tasks"} count={tasksRemaining} /> remaining!</h1>
     }
```

Comment:
5) Using the pluralize package and the `tasksRemaining` variable to display the correct message